server/routes/queries.js

```diff
@@ -933,7 +998,7 @@ router.delete("/testCleanup", async (req, res) => {
       await r.del(redisKeyeeRIS);
     }
     const query = `DROP TABLE IF EXISTS ${req.params.PROJECT_ID}_${eventCollection}`;
-    await client.query(query);
+    client.query(query);
```

Comment:
If this (`DELETE /testCleanup`) is a slow request that you don't want to wait for a response, then you should handle that client-side, not server-side.

Reply:
Will examine later as part of testing..